src/database/migrations/20220601181025-create-user.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('Users', {
+		id: {
+			allowNull: false,
+			autoIncrement: true,
+			primaryKey: true,
+			type: Sequelize.INTEGER,
+		},
+		firstName: {
+			type: Sequelize.STRING,
+			allowNull: false,
+		},
+		lastName: {
+			type: Sequelize.STRING,
+			allowNull: false,
+		},
+		email: {
+			type: Sequelize.STRING,
+			allowNull: false,
+      unique: true
+		},
+		password: {
+			type: Sequelize.STRING,
+		},
+		createdAt: {
```

Comment:
user needs to have a role, so userRole attribute is needed and it holds an integer value and not null